optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/impl/statistic/StatisticUtils.java

```diff
@@ -68,8 +69,12 @@ public static String getStandardDeviationString(double[] standardDeviationDouble
         StringBuilder standardDeviationString = new StringBuilder(standardDeviationDoubles.length * 9);
         // Abbreviate to 2 decimals
         // We don't use a local sensitive DecimalFormat, because other Scores don't use it either (see PLANNER-169)
+        DecimalFormatSymbols decimalFormatSymbols = DecimalFormatSymbols.getInstance();
```

Comment:
Changing the DecimalFormatSymbols.getInstance() of the entire JVM has lasting side effects.

1) Presume we have a benchmark server that generates the reports on the fly (based on previous benchmark runs - much like the aggregator). In parallel, it can get asked to generated 2 or more requests, with a different locale (the local is determined by the browser of the client, and is part of the HTTP request - the server's default local isn't relevant in this case). If that happens, there could be a race condition that the 1th request gets its number formatted in the locale of the 2th request.

2) Presume that the JVM app that embeds our benchmarker runs something else after it that uses DecimalFormatSymbols and doesn't explicitly set the default locale's setting for it (because it's the default even though we changed it).